tests/general/QuotesBearTest.py

```diff
@@ -97,3 +97,19 @@ def test_invalid_quotes(self):
                              '-"a string with double quotes!"\n'
                              "+'a string with double quotes!'\n"
                              " 'A single quoted string with \" in it'\n")
+
+        self.section['preferred_quotation'] = '"'
+        with execute_bear(self.uut, self.filename, self.double_quote_file,
+                          dependency_results=self.dep_results,
+                          force_preferred_quotation=True) as results:
+            res_list = list(results)
+            self.assertEqual(len(res_list), 1)
+            self.assertEqual(res_list[0].diffs[self.filename].unified_diff,
+                             '--- \n'
+                             '+++ \n'
+                             '@@ -3,4 +3,4 @@\n'
+                             ' Multiline string\n'
+                             " '''\n"
+                             ' "a string with double quotes!"\n'
+                             "-'A single quoted string with \" in it'\n"
+                             '+"A single quoted string with \\" in it"\n')
```

Comment:
could you also test a case with `'` as preferred quotes, and a string where no preferred quotes are inside? :+1: